src/sections/file/file-metadata/FileMetadata.tsx

```diff
@@ -123,7 +124,10 @@ export function FileMetadata({
                     format="YYYY-MM-DD"
                   />
                 ) : (
-                  DateHelper.toDisplayFormatYYYYMMDD(metadata.publicationDate)
+                  <time>
```

Comment:
With this logic if there is no `publicationDate` you will still render an empty <time/> tag, try to check if exits and  then render the tag